src/components/Navbar.vue

```diff
@@ -36,40 +36,62 @@
           <b-dropdown-item to="/about/team">
             Our Team
           </b-dropdown-item>
-          <b-dropdown-item
-            to="/about/publications">
-            Publications
-          </b-dropdown-item>
 
           <b-dropdown-item href="https://medium.com/@MonarchInit" target="_blank" rel="noopener noreferrer">
-            Blog
+            Disclaimer
           </b-dropdown-item>
 
-          <b-dropdown-item href="https://github.com/monarch-initiative" target="_blank" rel="noopener noreferrer">
-            GitHub Organization
+          <b-dropdown-divider/>
+
+          <b-dropdown-item to="/sources">
+            Data Sources
           </b-dropdown-item>
 
-          <b-dropdown-divider/>
+          <b-dropdown-item to="/sources">
+            Monarch API
+          </b-dropdown-item>
+
+          <b-dropdown-item to="/sources">
+            Releases
+          </b-dropdown-item>
 
           <b-dropdown-item href="https://archive.monarchinitiative.org/latest/" target="_blank" rel="noopener noreferrer">
             Data Downloads
           </b-dropdown-item>
+        </b-nav-item-dropdown>
+
+        <b-nav-item-dropdown text="Documentation">
+          <b-dropdown-item to="/about/publications">
+            Publications
+          </b-dropdown-item>
 
-          <b-dropdown-item to="/sources">
-            Data Sources
+          <b-dropdown-item href="https://github.com/monarch-initiative" target="_blank" rel="noopener noreferrer">
+            GitHub Organization
+          </b-dropdown-item>
+
+          <b-dropdown-item href="https://github.com/monarch-initiative" target="_blank" rel="noopener noreferrer">
+            Guidelines
+          </b-dropdown-item>
+
+          <b-dropdown-item href="https://github.com/monarch-initiative" target="_blank" rel="noopener noreferrer">
+            Phenotype Ontologies
```

Comment:
@iimpulse This sub-menu item 'Phenotype Ontologies' and the next one 'Project' are actually part of the same item "Phenotypes Ontologies Project" - that was my mistake for putting them in two different lines when I made the mockup. Unfortunately, I did not realize that the line break would be interpreted as separate menu items. This sub-menu item, and the previous one ("Phenotype Curation Guidelines") are probably going to be too long to stay in one line with the current font size. We will need to reduce the font size, or make the drop-down menu column wider.